neurodamus/core/configuration.py

```diff
@@ -949,13 +949,7 @@ def _check_restore(config: _SimConfig, run_conf):
 
 
 @SimConfig.validator
-def _coreneuron_params(config: _SimConfig, run_conf):
-    buffer_size = run_conf.get("ReportingBufferSize")
-    if buffer_size is not None:
-        assert buffer_size > 0
-        log_verbose("ReportingBufferSize = %g", buffer_size)
-        config.corenrn_buff_size = int(buffer_size)
-
+def _coreneuron_params(config: _SimConfig, _run_conf):
```

Comment:
why this change? It is not mentioned in the description or the issue

Reply:
Done, I have clarified in the PR description.